tinygrad/renderer/cstyle.py

```diff
@@ -45,7 +45,7 @@ def render_cast(self, x:List[str], var_dtype:DType, bitcast=False) -> str:
   def render_const(self, x:Union[float,int,bool], var_dtype) -> str:
     if math.isnan(x): val = "NAN"
     elif math.isinf(x): val = ("-" if x < 0 else "") + "INFINITY"
-    else: val = f"{float(x)}f" if dtypes.is_float(var_dtype) else f"{int(x)}" if dtypes.is_int(var_dtype) else f"{bool(x)}".lower()
+    else: val = f"{float(x)}" if var_dtype == dtypes.float64 else f"{float(x)}f" if dtypes.is_float(var_dtype) else f"{int(x)}" if dtypes.is_int(var_dtype) else f"{bool(x)}".lower()  # noqa: E501
```

Comment:
No new lines with noqa E501

Reply:
Thanks for reviewing. Removed the noqa E501 exception.